packages/payment-integration-api/src/payment/payment-actions.ts

```diff
@@ -0,0 +1,53 @@
+import { Action } from '@bigcommerce/data-store';
+
+import { LoadOrderAction } from '../order';
+
+import PaymentResponseBody from './payment-response-body';
+
+export enum PaymentActionType {
```

Comment:
Do we need to move this across?

Reply:
yes, we use it in the SubmitPaymentAction